tests/Unit/Pool/accelerator_get_compute_unit_count_api.cpp

```diff
@@ -0,0 +1,28 @@
+// XFAIL: Linux
+// RUN: %hc %s -I%hsa_header_path -L%hsa_library_path -lhsa-runtime64 -o %t.out && %t.out
```

Comment:
could you change the macro so now HSA header & HSA runtime lib is needed at compile time?